extensions/amp-video/0.1/amp-video.js

```diff
@@ -62,6 +63,21 @@ const ATTRS_TO_PROPAGATE_ON_LAYOUT = ['loop', 'preload'];
 const ATTRS_TO_PROPAGATE =
     ATTRS_TO_PROPAGATE_ON_BUILD.concat(ATTRS_TO_PROPAGATE_ON_LAYOUT);
 
+
+/**
+ * @param {!Element} ctx
+ * @param {string} src
+ */
+function createPosterImage(ctx, src) {
```

Comment:
Spoke in person about this item, adding a comment regarding the planned reuse makes it more clear why this is externalized.

Reply:
Inlined into component method as re-use is not required yet, so I'd rather let the code reflect the current state :)